packages/main/src/Table.hbs

```diff
@@ -12,9 +12,18 @@
 
 	<!-- rows -->
 	{{#each ctr.rows}}
-		<div style="width: 100%"
-		>
-			<slot name="{{this._individualSlot}}"></slot>
-		</div>
+	<div style="width: 100%">
+    <slot name="{{this._individualSlot}}"></slot>
+	</div>
 	{{/each}}
+
+	{{#unless ctr.rows.length}}
+		{{#if ctr.showNoData}}
+			<div style="width: 100%">
+				<div id="noData" class="{{classes.main}} sapWCTableNoData">
```

Comment:
Remove {{classes.main}}, rename sapWCTableNoData to sapWCTableNoDataRow and the changes in the CSS are described in another comment.